### PR TITLE
vscode: start gzclient GUI for Gazebo simulation

### PR DESCRIPTION
**Pull request by julianoes**
@dagar I think it's a nice default to start the Gazebo GUI.

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
                     "GAZEBO_MODEL_PATH": "${workspaceRoot}/Tools/sitl_gazebo/models"
                 }
             },
-            "command": "gzserver --verbose ${workspaceRoot}/Tools/sitl_gazebo/worlds/iris.world",
+            "command": "gzserver --verbose ${workspaceRoot}/Tools/sitl_gazebo/worlds/iris.world & gzclient",
```
Comment (dagar) [inline .vscode/tasks.json]: I wonder if these should be separate tasks?

**Comment by dagar** [comment]
Yes that certainly makes for a better initial experience. I'll readd a secondary headless option.

File: .vscode/tasks.json (hunk 1 above)
Comment (julianoes) [reply .vscode/tasks.json]: I was wondering too but didn't know how to do that.

Comment (dagar) [reply .vscode/tasks.json]: I'll take a pass at it when adding a standalone headless option.

**Comment by hamishwillee** [comment]
FWIW Gazebo doesn't work on Windows. I guess neither does vscode.

**Pull request merged**